test/1.2/statements/T_if_ambiguity.dml

```diff
@@ -0,0 +1,60 @@
+/*
+  © 2023 Intel Corporation
+  SPDX-License-Identifier: MPL-2.0
+*/
+dml 1.2;
+
+device test;
+
+// This tests both that GCC doesn't emit Wparantheses and the behavior of
+// the DML parser.
+// Although GCC emitting Wparantheses could be considered desirable for the
```

Comment:
`s/Wparantheses/Wparentheses/`

Perhaps warrants adding "parentheses" to the wordlist

Reply:
Iiii suspect it would blow up half of Simics base ;)

or maybe it's just me who makes that mistake